tests/helpers/test_many.ts

```diff
@@ -73,7 +73,7 @@ export const makeTestMany = <M extends string>(
   const testFns = ['fails', 'only', 'concurrent', 'skip', 'todo']
 
   testFns.forEach((fn) => {
-    testBundlers[fn] = ((...args) => testBundlers(...args, test[fn])) as TestMany<M>
+    testBundlers[fn] = ((...args) => testBundlers(...args, testAPI[fn])) as TestMany<M>
```

Comment:
This does not really mattter, but it is cleaner to use the testAPI that is supplied to the `makeTestMany()` which is `test` instead of using `test` directly.